packages/dd-trace/test/instrumenter.spec.js

```diff
@@ -352,4 +353,60 @@ describe('Instrumenter', () => {
       })
     })
   })
+
+  describe('with plugins disabled via disabledPlugins configuration option', () => {
+    describe('enable', () => {
+      it('should not patch plugins disabled from configuration option', () => {
+        instrumenter._disabledPlugins = new Set(['express-mock', 'http'])
```

Comment:
The environment variable should be used instead of the private property, and can be reset in an `afterEach`.

Reply:
added a beforeEach / afterEach block so that the tests could use an environment variable explicitly, though this requires the beforeEach block to re-instantiate the instrumenter each time. Updated tests to remove private property. Also added a more robust test for plugins that should be patched.